test/pytest/test_plot_model.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+from pathlib import Path
+
+import pytest
+from tensorflow.keras.models import model_from_json
+
+import hls4ml
+from hls4ml.utils.plot import plot_model
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+test_root_path = Path(__file__).parent
+example_model_path = (test_root_path / '../../example-models').resolve()
+
+
+@pytest.fixture(scope='module')
+def load_jettagging_model():
+    model_path = example_model_path / 'keras/KERAS_3layer.json'
+    with model_path.open('r') as f:
+        jsons = f.read()
+    model = model_from_json(jsons)
+    model.load_weights(example_model_path / 'keras/KERAS_3layer_weights.h5')
+    return model
+
+
+@pytest.fixture(scope='module')
+def convert(load_jettagging_model):
+    model = load_jettagging_model
+    config = hls4ml.utils.config_from_keras_model(model, granularity='name', backend='vitis')
+    hls_model = hls4ml.converters.convert_from_keras_model(
+        model,
+        hls_config=config,
+        output_dir=str(test_root_path / 'hls4mlprj_qkeras'),
+        part='xcu250-figd2104-2L-e',
+    )
+    hls_model.compile()
+    return hls_model
+
+
+def test_plot_model(convert):
+    # Test the plot_model function with a sample model.
+    output_file = "model_plot.png"
```

Comment:
It's only minor, but hls4ml tends to use single quotes instead double quotes.

Reply:
I fixed the quotes and the italic font.